.github/workflows/build-snap.yml

```diff
@@ -48,14 +48,36 @@ jobs:
           sudo snap connect openstack:dot-local-share-juju
           sudo snap connect openstack:dot-config-openstack
           sg snap_daemon "openstack.sunbeam -v cluster bootstrap --accept-defaults"
-          sg snap_daemon "openstack.sunbeam enable orchestration"
           sg snap_daemon "openstack.sunbeam cluster list"
+          sg snap_daemon "openstack.sunbeam enable orchestration"
           sg snap_daemon "openstack.sunbeam enable loadbalancer"
-          sg snap_daemon "openstack.sunbeam -v configure -a"
-          sg snap_daemon "openstack.sunbeam -v launch"
+          sg snap_daemon "openstack.sunbeam enable dns --nameservers=testing.github."
+          sg snap_daemon "openstack.sunbeam enable telemetry"
+          sg snap_daemon "openstack.sunbeam enable observability"
+          sg snap_daemon "openstack.sunbeam enable vault"
+          sg snap_daemon "openstack.sunbeam enable secrets"
+          sg snap_daemon "openstack.sunbeam enable caas"
+          sg snap_daemon "openstack.sunbeam configure -a"
```

Comment:
Configure will also run configure caas. You can keep the `sunbeam configure caas`, but it's not needed and will be fast.

Reply:
Removed `sunbeam configure caas`